dev-haskell/network/files/network-3.1.2.6-size_t.patch

```diff
@@ -0,0 +1,93 @@
+From 206fb152261f10c9b55543103526e13d25377169 Mon Sep 17 00:00:00 2001
```

Comment:
Add a link to the upstream PR/commit at the top of the patch please

Reply:
Done